cargo/private/cargo_build_script.bzl

```diff
@@ -423,18 +420,25 @@ def _cargo_build_script_impl(ctx):
     # Pull in env vars which may be required for the cc_toolchain to work (e.g. on OSX, the SDK version).
     # We hope that the linker env is sufficient for the whole cc_toolchain.
     cc_toolchain, feature_configuration = find_cc_toolchain(ctx)
-    linker, link_args, linker_env = get_linker_and_args(ctx, "bin", cc_toolchain, feature_configuration, None)
+    linker, _, link_args, linker_env = get_linker_and_args(ctx, "bin", toolchain, cc_toolchain, feature_configuration, None)
     env.update(**linker_env)
     env["LD"] = linker
     env["LDFLAGS"] = " ".join(_pwd_flags(link_args))
 
-    # MSVC requires INCLUDE to be set
-    cc_c_args, cc_cxx_args, cc_ar_args, cc_env = get_cc_compile_args_and_env(cc_toolchain, feature_configuration)
-    include = cc_env.get("INCLUDE")
-    if include:
-        env["INCLUDE"] = include
+    # Defaults for cxx flags.
+    env["CC"] = "${{pwd}}/{}".format(ctx.executable._fallback_cc.path)
```

Comment:
nit: can we define these inside the build_script_runner executable to avoid passing extra env vars we don't need?

Reply:
https://github.com/bazelbuild/rules_rust/pull/3665#discussion_r2581771403 for this reason I think this is harder than it's worth or spreads out configuration definitions.